public/js/report.js

```diff
@@ -975,40 +975,45 @@ var SideBar = React.createClass({
     },
 
     /**
+     * Dynamically create form and submit.
+     */
+    postForm: function ($a, sequence_ids) {
```

Comment:
The only use of `$a` in the entire function is to get database ids. Why not pass database ids as an argument instead?